### ISSUE TITLE
Problematic focus indicator for form controls

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3126]
The available sliders (`<input>` with type="range") rely on color only to indicate their  focused state, when reached by pressing Tab or Shift+Tab. The label text updates its  font color from gray (#999) to yellow `#D9DA6D`. The change is subtle and may not  be noticeable for many users, since the contrast ratio between the two colors is very  low, at 1.48:1. 

Additionally, such styling is not persistent when using Windows Contrast Themes and  no other indication of the focus state exists.  

**Suggested Solution**
Restore the focus outline and ensure that a highly visible indication of the focused state exists. See: Missing Focus Indicator on Many Controls. 

Severity: 2

**Comment by engbergandreas** [comment]
Fixed with https://github.com/OpenSpace/OpenSpace/commit/e3f5b83f7476c8507dccd45ab98e4731504af532?

**Comment by alexanderbock** [comment]
I'd think so

**Comment by WeirdRubberDuck** [comment]
I would have said yes, but since we changed the focus color to white, the focus indicator for the numeric sliders is very ambiguous:

This is how it looks like when focused: 
![Image](https://github.com/user-attachments/assets/b2459f52-4b49-4640-9846-5a3acee54b46)

VS non-focused: 
![Image](https://github.com/user-attachments/assets/2e1796dc-5b7d-4973-87a8-641af2898d27)

i.e. the only noticable difference is the value-hint-tooltip thing

But either way, mayb close this one and create a new issue since the description is no longer applicable?

**Comment by alexanderbock** [comment]
Yea :+1:

**Comment by alexanderbock** [comment]
Closed in favor of #3710
